### PR TITLE
Logging information improved

### PR DESCRIPTION
**Pull request by M1ch43lV**
To make a more readable output, specially concering the incoming Telegram messages and the BTC up-/downtrend confirmation.
This update tries to avoid questioning the 3CQS signal processing, e.g. in case of missing signals and to allow more traceability and transparency of the bot working function.

### CODE REVIEW

File: 3cqsbot.py [hunk 1]
```diff
@@ -281,9 +282,10 @@ async def my_event_handler(event):
         logging.info("Bot stopped - no new signals processed")
     else:
 
-        logging.debug("New signals incoming...")
+        logging.info("New 3CQS signals incoming...")
 
         tg_output = tg_data(parse_tg(event.raw_text))
+        logging.info("TG msg: "+ str(tg_output))
```
Comment (TBMoonwalker) [inline 3cqsbot.py]: @M1cha3l-2022  Do we really want this to be info? This will mess up the info logging. Please make it debug.

File: signals.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import asyncio
 import math
 import re
+import logging
```
Comment (TBMoonwalker) [inline signals.py]: @M1cha3l-2022 not needed - it is part of the class init (see line 15)

File: signals.py [hunk 3]
```diff
@@ -72,6 +73,10 @@ def topcoin(self, pairs, rank):
         market = self.cgvalues(rank)
 
         self.logging.debug(self.cgvalues.cache_info())
+        if isinstance(pairs, str):
```
Comment (TBMoonwalker) [inline signals.py]: @M1cha3l-2022  is instance(pairs, str) is already in the else statement. Logging should be there. First log output can go in line 93 and second one in line 82 (each after pairlist attribute)

**Review by M1ch43lV — COMMENTED** [review]
Suggested changes implemented :-)

**Pull request merged**